docs/design/alias-configuration.md

```diff
@@ -52,4 +69,56 @@ In this flow
+- Crypto Transfer Transaction (without auto creation): to be called with EVM address (the account with this address is already auto lazy created)
+  - We need to extract the public key from signature with extractSig
+  - After that to use the extracted key for recovering the public address (recoverAddressFromPubKey)
+  - Recovered address must match with the alias (public address of the eth account)
+  - If the recovered address match with the address stored in the alias we have to update the key property of the account and store it in the ledger
+  - With the above step the process of lazy creation will be finished
+- Crypto Transfer Transaction to ECDSA key alias should create an account with its alias field set to the ECDSA key, populate the alias map with the EVM address calculated based on the ECDSA key, and the key field set with the value of the ECDSA key (currently implemented) and another Crypto Transfer Transaction to the EVM address corresponding to the ECDSA key should transfer to the account created in the previous transaction
+- Crypto Create Transaction with ED25519 key alias should create an account with its alias and key fields set to the ED25519 key. Attempting another Crypto Create Transaction with the same ED25519 key alias should fail
+- Crypto Create Transaction with ECDSA key alias:
+    
+**Always validate the 0.0.ECDSA alias format is missing from the alias map as well as the calculated evm_address alias format**
+    
+  1. Create an account with its alias and key fields set to the ECDSA key, and populate the alias map with the EVM address calculated based on the ECDSA key.
+      - Account with:
+          - hedera id: 0.0.20
+          - admin key: ECDSA key
+          - alias: 0.0.ECDSA key
+      - AliasManager map with entries:
+          - calculated EVM_address alias based on the ECDSA key → hedera id
+          - 0.0.ECDSA key → hedera id
+  
+  2. Attempting another Crypto Create Transaction with the same ECDSA key alias or EVM address corresponding to that key should fail
+    
+- Crypto Create Transaction with EVM address alias
+    1. Create an account with its alias field set to the EVM address and its key will be empty
+        - Account with:
+            - hedera id: 0.0.20
+            - admin key:
+            - alias: EVM_address alias
+        - AliasManager with:
+            - map with the EVM_address alias based on the ECDSA key → hedera id
+    2. a) When a signed transaction referencing the account by the EVM_address alias is received:
+        - Account with:
+            - hedera id: 0.0.20
+            - admin key: ECDSA key
+            - alias: 0.0.ECDSA key
+        - AliasManager with:
+            - map with the EVM_address alias based on the ECDSA key → hedera id
+            - map with the 0.0.ECDSA key alias → hedera id
+    
+    2. b) Don’t overwrite the alias field, instead add logic during the map rebuilding
```

Comment:
Can you clarify why we wouldn't override the alias field stored tot eh public key once we get it?
Since we're trying to map into existing flow we should ensure that the only case where an alias on account (not the map) is an evmAddress is when it's a Hollow account i.e. in between state and waiting for a subsequent transaction to complete the onboarding process

Reply:
The 2.b) scenario was written down as an alternative to 2.a) i.e. if we are not fine with an approach in which we update the account alias. In 1. we create an account with alias set to the EVM address and empty key and then in 2.a) we set the key and update the alias to have the ECDSA key value.

We were not sure if updating the account alias is ok as an approach but if that is fine we will prefer to go with the 2.a) option.

Reply:
We should be updating the account alias if it is the evm address and we get a publickey in the signature map that maps to the evm address

Reply:
`2. a)` and `2. b)` were two options that we needed to discuss and choose from. 
Now that we have chosen option `2. a)` I think we should delete `2. b)` from this document for clarity and leave only `2.` with added description for the update of the account alias.

Reply:
@MiroslavGatsanoga  and @stoqnkpL Final note on this.
Reverting my above note.
For lazy create we should not update the alias from evmAddress to publicKey.
We should only set the alias as the publicKey when a user does auto-create and submits the public key.
I'll also update the HIP with this